content/History.md

```diff
@@ -1,10 +1,15 @@
+## Historical Overview
 
 _Franz Sigg_
 
-The ConRes Test target came out of a discussion Franz Sigg had with Dr. Ed. Granger at a lunch table at RIT in the fall of 1999. We were discussing a thesis topic for Eliot Harper. After several design versions of the target by Franz Sigg, Eliot Harper finished his RIT thesis in Dec. 2000 with the title <q>An Investigation Into the Relationship Between Contrast and Resolution of a Printing System Using the RIT Contrast Resolution Test Target</q> He also was the graduate student paper winner at Taga 2001, with a paper with the same title but a more compact description of the ConRes methodology. In 2003, Deepak Dubay wrote an RIT student TAGA paper with the title <q>Relationship between Contrast Resolution Volume (CRV) and Modulation Transfer Function (MTF)</q> which preceded his thesis <q>A Comparative Analysis between the RIT Contrast Resolution Test Target And the Gutenberg Test Target</q> that was published in 2005. In 2006 Franz Sigg wrote a paper for the RIT TestTargets 6 publication with the title <q>Testing for Resolution and Contrast</q> The topic was also taught to students at RIT.
+The ConRes test target came out of a discussion Franz Sigg had with Dr. Ed. Granger at a lunch table at RIT in the fall of 1999 related to the thesis topic for Eliot Harper, who was working on his Masters at the time. This discussion led to first attempts to making what eventually becomes the Contrast-Resolution test target that is familiar to many today, and for which earliest research uses date back the completed thesis work of Eliot Harper (2000) title "An Investigation Into the Relationship Between Contrast and Resolution of a Printing System Using the RIT Contrast Resolution Test Target" and his winning entry of the 2001 Technical Association of the Graphic Arts graduate research paper of the same title.
 
-There are some 29 implementations of the Target, each one hand programmed in PostScript by Franz Sigg. All versions use vector graphics.
+This was only the start of many extensive research endeavors, involving additional RIT student work, including Deepak Dubay's (2003) RIT student TAGA paper on the "Relationship between Contrast Resolution Volume (CRV) and Modulation Transfer Function (MTF)" leading to his thesis (2005) on "A Comparative Analysis between the RIT Contrast Resolution Test Target And the Gutenberg Test Target".
+
+In 2006 Franz Sigg wrote a paper for the RIT TestTargets 6 publication with the title "Testing for Resolution and Contrast" The topic was also taught to students at RIT.
+
+To date, there are some 29 implementations of the target, all hand programmed in PostScript by Franz Sigg to elicit pixel perfect vector-based detail and tone that have made it possible to measure the limitations and correlation extents of those aspects in print and other imaging domains, where otherwise they would have remained inconceivable or untenable to even explore.
```

Comment:
@FranzSigg 29 implementations to date, right?

Reply:
Actually we are now working on version 35. 
But some of these versions vary very little from the previous one. Others are major and vary a lot.

Reply:
In the document, I have not shown all the versions, because for many the changes would be too small or invisible, because they are just a better way of programming.

Reply:
Good to know, and absolutely, from a reader's perspective it makes more sense to condense them that way — so I take it 29 ~= 35 in that regard.